webapp/src/ts/modules/reports/reports-content.component.html

```diff
@@ -97,27 +97,28 @@
                     <p>{{selection.formatted.kujua_message ? message.to : message.message}}</p>
                   </li>
                 </ul>
-                <span class="task-state" [innerHTML]="task | state"></span>
+                <span class="task-state" [innerHTML]="task | state | async"></span>
               </li>
               <li class="clear"></li>
             </ul>
           </li>
         </ul>
-        <div class="scheduled-tasks" *ngIf="selection.formatted.scheduled_tasks_by_group?.length">
+        <div class="scheduled-tasks" *ngIf="selection?.formatted?.scheduled_tasks_by_group?.length">
           <h2 translate>tasks</h2>
           <ul>
             <li *ngFor="let group of selection.formatted.scheduled_tasks_by_group">
               <h3>{{group.name}}</h3>
               <p>
-                <span ng-hide="group.loading">
+                <span *ngIf="!group.loading">
+                  <!--TODO migrate these functions-->
                   <a class="btn btn-link" ng-click="edit(selection, group)" translate>Edit</a>
```

Comment:
Just to double check, these ng-click and ng-show will be migrated in future PR?

Reply:
Yes! It's the 11th TODO in the angular migration doc. 
These are super low priority because they're broken on master: https://github.com/medic/cht-core/issues/6385
I reported this bug this April and it takes almost 1 year to get scheduled in the **next** release :D